src/Components/Typography/styles.ts

```diff
@@ -0,0 +1,144 @@
+import "@fontsource/mulish";
+import styled from "styled-components";
+import { TypographyTypes } from "./consts";
+
+export const StyledTypography = styled.div<{ type: string }>`
```

Comment:
the type should be `TypographyTypes` and not `string`, this is exactly why we use enums in the first place.

Reply:
done